gateway/src/services/config-manager-v2.ts

```diff
@@ -3,29 +3,60 @@ import { ValidateFunction } from 'ajv';
 import fs from 'fs';
 import path from 'path';
 import yaml from 'js-yaml';
+import * as migrations from '../../conf/migration/migrations';
```

Comment:
Change this line after moving the config migration file. Besides, as a reminder, importing the conf directory directly doesn't work.

Reply:
I thought `"copy-files": "copyfiles -a 'src/**/schema/*.json' 'test/services/data/**/*.*' 'src/templates/*.*' dist",` in the package.json handles that. Anyway, I'll update it not to copy over files from the conf directory since we've moved the template and schema to the project src